src/core/BufferAttribute.js

```diff
@@ -22,7 +23,10 @@ class BufferAttribute {
 		this.array = array;
 		this.itemSize = itemSize;
 		this.count = array !== undefined ? array.length / itemSize : 0;
-		this.normalized = normalized === true;
+
```

Comment:
I think this is not needed because the original line will perform the same steps in the setter?